vite-project-kabinet/src/main.jsx

```diff
@@ -11,7 +11,8 @@ ReactDOM.createRoot(root).render(
   <BrowserRouter>
     <Routes>
       <Route path="/" element={<App />}/>
-      <Route path="/clanek" element={<CardDetail/>}/>
+      <Route path="/esej/:title" element={<CardDetail/>}/>
```

Comment:
Slova `esej` a `texty` bude lepší když uložíš do konstanty a všude, kde to používáš (aktuálně dvě místa) použiješ tu konstantu. Bude to jednodušší, když to budeš chtít někdy změnit (třeba se rozhodneš, že když "texty" jsou množné číslo, tak by tak měly být i "eseje". Obecně je dobré nemít duplikované řetěze jako tyto ale uložit je do proměnné a při změně se pak na nic nezapomene.